core/src/avm2/globals/flash/utils.rs

```diff
@@ -386,6 +387,8 @@ fn describe_internal_body<'gc>(
         if !ns.is_public() {
             continue;
         }
+        let slot_metadata_table = vtable.slot_metadata_table();
```

Comment:
instead of exposing the tables via `Ref`, how about encapsulating with
```
let trait_metadata = vtable.get_metadata_for_slot(slot_id);
```
?

Reply:
I prefer it the way it currently is.